en/Plugins/Search.md

```diff
@@ -1,82 +1,98 @@
+| `ignore-case:`  | Case-insensitive match.<p/>Example: `ignore:case:ikea`.                                                                                                                                                              |
+| `tag:`          | Find tag in file.<p/>Example: `tag:#work`.<p/>**Note**: Since `tag:` ignores matches in code blocks and in non-Markdown content, it's often faster and more accurate than a normal full-text search for `#work`.     |
+| `line:`         | Find matches on the same line.<p/>Example: `line:(mix flour)`.                                                                                                                                                       |
+| `block:`        | Find matches in the same block.<p/>Example: `block:(dog cat)`.<p/>**Note**: Since `block:` requires Search to parse the Markdown content in every file, it can cause your search term to take longer time to finish. |
+| `section:`      | Find matches in the same section (text between two headings).<p/>Example: `block:(dog cat)`.                                                                                                                         |
+| `task:`         | Find matches in a [[Format your notes#Task list\|task]] on a block-by-block basis.<p/>Example: `task:call`.                                                                                                          |
+| `task-todo:`    | Find matches in an *uncompleted* [[Format your notes#Task list\|task]] on a block-by-block basis.<p/>Example: `task-todo:call`.                                                                                      |
+| `task-done:`    | Find matches in a *completed* [[Format your notes#Task list\|task]] on a block-by-block basis.<p/>Example: `task-done:call`.                                                                                         |
 
-For example, if you write:
+## Use regular expressions in search terms
 
-<pre><code>```query
-embed OR search
-```</code></pre>
+A regular expression is a set of characters that describe a text pattern. To use regular expressions in your search term, surround the expression with forward slashes (`/`).
 
-You should see this embedded search view (note: it doesn't work on [[Introduction to Obsidian Publish|Obsidian Publish]] as of 2020/01/18):
+- `/\d{4}-\d{2}-\d{2}/` matches an ISO 8601 date, such as 2022-01-01.
 
-```query
-embed OR search
-```
+You can even combine regular expressions with search operators:
+
+- `path:/\d{4}-\d{2}-\d{2}/` returns files with a date in the file path.
 
-## Search syntax
+For more information on how to write regular expressions, refer to [Regular expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
```

Comment:
Love the link to Mozilla docs! It's unlikely that we can teach people how to use RegExp within our docs haha